### DISCUSSION TITLE
The possibility of sponsorship of developers from Kyrgyzstan or CIS countries

### DISCUSSION THREAD

**Discussion by warioddly** [community/community#118769] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I am a developer from Kyrgyzstan. We already have quite a lot of developers, but for the CIS countries (except Uzbekistan) we still do not have sponsorship support. Would you like to know when this will be a sponsorship opportunity? (That is, we still do not have the opportunity to add a sponsor button in our profile)

**Comment by anjuan** [comment, marked as answer]
Hi @warioddly, I'm the Engineering Manager for GitHub Sponsors. I absolutely want GitHub Sponsors to be available everywhere people use GitHub, and we've made [consistent progress towards that goal](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#supported-regions-for-github-sponsors). Unfortunately, Sponsors hasn't made it to Kyrgyzstan yet due to a variety of reasons. Although I can't provide a timeline, I really hope we can offer Sponsors in your nation and the other CIS countries.

In the meantime, if you use other funding platforms (Patreon, Tide Lift, Buy Me a Coffee, etc.), GitHub supports [adding a funding button to your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository).

**Reply by warioddly** [reply]
It would be cool to know when this will be possible (approximate time). But in any case, @anjuan thank you very much, we look forward to it)
